### PR TITLE
removed admin endpoint from other services and updated hc script

### PR DESCRIPTION
**Pull request by sockterrier**

### CODE REVIEW

File: migrations/src/zenservicemigration/migrations/update_zing_connector_config_for_compact_third.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+Further update zing-connector config to support compact-metrics.
+"""
+
+from __future__ import print_function
+
+import sys
+
+import servicemigration as sm
+
+version = "7.0.20"
```
Comment (jpeacock-zenoss) [inline migrations/src/zenservicemigration/migrations/update_zing_connector_config_for_compact_third.py:20]: The version should be "7.1.0"

**Review by bharatZen — APPROVED** [review]

File: migrations/src/zenservicemigration/migrations/update_zing_connector_config_for_compact_third.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    changed = False
+
+    if answering.script != new_script:
+        answering.script = new_script
+        print("Updated 'answering' healthcheck script for zing-connector service.", file=sys.stderr)
+        service.healthChecks = [answering]
+        changed = True
+
+    for svc in other_services:
+        changed = changed or remove_endpoint_and_fix_hc(svc)
```
Comment (jpeacock-zenoss) [inline migrations/src/zenservicemigration/migrations/update_zing_connector_config_for_compact_third.py:58]: The `or` has short-circuiting behavior.  When `changed` is True, the `remove_endpoint_and_fix_hc` function will not be called.

**Review by jpeacock-zenoss — CHANGES_REQUESTED** [review]

File: migrations/src/zenservicemigration/migrations/update_zing_connector_config_for_compact_third.py (hunk 2 above)
Comment (sockterrier) [reply migrations/src/zenservicemigration/migrations/update_zing_connector_config_for_compact_third.py:58]: ahh, shoot. thanks, will fix, one sec.

File: migrations/src/zenservicemigration/migrations/update_zing_connector_config_for_compact_third.py (hunk 1 above)
Comment (sockterrier) [reply migrations/src/zenservicemigration/migrations/update_zing_connector_config_for_compact_third.py:20]: ok

**Review by jpeacock-zenoss — APPROVED** [review]

**Pull request merged**